src/brackets.js

```diff
@@ -358,8 +358,8 @@ define(function (require, exports, module) {
         }
         
         // Localize MainViewHTML and inject into <BODY> tag
-        $("body").html(Mustache.render(MainViewHTML, Strings));
-        
+        $("body").html(Mustache.render(MainViewHTML, { shouldAddAA: (brackets.platform === "mac"), Strings: Strings }));
```

Comment:
This doesn't work right now, you need to add the `Strings.` namespace in `main-view.html` to the template vars there.
But read my other comment first.